### PR TITLE
46 update docker commands and remove conda

### PR DESCRIPTION
**Pull request by Fr3dMer**
# See issue #46
- [x] Fix janky docker command in readme
- [x] Remove conda reliance from docker 
- [x] Solve Requirements.txt vs pyproject.toml ambiguity (Amy solved in #50) 
- [x] Improve commenting in Dockerfile 
- [ ] Stretch goal: add jenkins file 

Do not have time to add jenkins file, if anyone else feels like they can please feel free to help

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+numpy==1.26.3
+pandas==2.2.0
+pytest==7.4.3
+Requests==2.31.0
+responses==0.24.1
+SQLAlchemy==2.0.25
+coverage == 7.2.2
```
Comment (AbiHaddon) [inline docs/requirements.txt:7]: Minor issue - so not required. It is suggested that the more suitable syntax for coverage is **coverage >= 7.2.2'**
'This specifies that any version of coverage greater than or equal to 7.2.2 is acceptable. This allows for patch updates to be installed automatically, ensuring compatibility while still benefiting from bug fixes and improvements in newer versions.'

**Review by Emma-tho — APPROVED** [review]
Looks good!

**Review by AmyGrimwood — APPROVED** [review]
Looks good,
Cant see any PEP8 conflicts apart from some of the file names being capitalised.
Quite a few of our files are capitalised which looks like is PEP8 conflict should we change these?

File: docs/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+numpy==1.26.3
+pandas==2.2.0
+pytest==7.4.3
+Requests==2.31.0
```
Comment (AbiHaddon) [inline docs/requirements.txt:4]: is requests meant to have a lowercase r?

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "PannelAppDB"
+authors = [{name = "Frederick Mercer", email = "frederick.mercer@postgrad.manchester.ac.uk"}{name = "Abigail Haddon", email = "abigail.haddon@postgrad.manchester.ac.uk"}{name = "Amy Grimwood", email = "amy.grimwood@postgrad.manchester.ac.uk"}]
+dynamic = ["version", "description"]#
```
Comment (AbiHaddon) [inline pyproject.toml:8]: is there meant to be a '#' after ["version", "description"]**#** ??

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+dependencies = ["numpy==1.26.3",
+                "pandas==2.2.0",
+                "pytest==7.4.3",
+                "Requests==2.31.0",
+                "responses==0.24.1",
+                "SQLAlchemy==2.0.25",
+                "coverage == 7.2.2"]
```
Comment (AbiHaddon) [inline pyproject.toml:15]: "coverage == 7.2.2" - remove extra spacing from around operator

**Review by AbiHaddon — DISMISSED** [review]

File: Dockerfile [hunk 5]
```diff
@@ -1,11 +1,17 @@
 # syntax=docker/dockerfile:1
```
[the remark this answers is not in the document]
Comment (Fr3dMer) [reply Dockerfile:2]: Removed docker syntax definer as plays little role here and added header

File: docs/requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fr3dMer) [reply docs/requirements.txt:7]: Part of python standard lib so didn't initially think it's worth it

File: docs/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+numpy==1.26.3
```
[the remark this answers is not in the document]
Comment (Fr3dMer) [reply docs/requirements.txt:1]: Header added

File: pyproject.toml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
```
[the remark this answers is not in the document]
Comment (Fr3dMer) [reply pyproject.toml:4]: Dont think it needs a header https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/

File: docs/requirements.txt (hunk 1 above)
Comment (Fr3dMer) [reply docs/requirements.txt:7]: Pinned exact version to protect against future updates introducing incompatibilities with other libs

File: pyproject.toml (hunk 3 above)
Comment (Fr3dMer) [reply pyproject.toml:8]: Good catch, removed

File: pyproject.toml (hunk 4 above)
Comment (Fr3dMer) [reply pyproject.toml:15]: fixed

File: pyproject.toml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+[build-system]
```
[the remark this answers is not in the document]
Comment (Fr3dMer) [reply pyproject.toml:1]: Dont think it needs a header https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/

**Comment by Fr3dMer** [comment]
> Looks good, Cant see any PEP8 conflicts apart from some of the file names being capitalised. Quite a few of our files are capitalised which looks like is PEP8 conflict should we change these?

your right theres quite a few and also a typo in `PannelAppDB.py` I'm happy to do this at the end for finishing touches?

**Review by AmyGrimwood — COMMENTED** [review]
Looks good, thank you for resolving our comments

**Pull request merged**